packages/dds/merge-tree/src/client.ts

```diff
@@ -113,18 +113,18 @@ export class Client {
      * It is used to get the segment group(s) for the previous operations.
      * @param count - The number segment groups to get peek from the tail of the queue. Default 1.
```

Comment:
Not related to your change, but I wouldn't know without looking at the code what order the returned SegmentGroups were in. It would be helpful to update the comment, optionally.